src/main/webapp/css/user-page.css

```diff
@@ -36,12 +36,14 @@
 
 /* Give the message body a margin so the text
    isn't too close to the border. */
-.message-body {
-  margin: 5px;
```

Comment:
Why delete this margin?

Reply:
I thought it does the same as .message-body img {

Reply:
Not exactly--this adds a margin to every message, because it selects everything with the .message-body css class. The .message-body img only styles image tags, so it doesn't change just plain messages.

If you look at before this change, there's a little bit of space between the border and the message, but with this deleted the words are right up against the edges.